index.html

```diff
@@ -0,0 +1,390 @@
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__1.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__2.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="pricing-plan">
+            <div class="container">
+                <div class="main__name">
+                    <p class="main__name__title">Pricing Plan</p>
+                    <p class="main__name__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti quos porro,<br> ipsa sed 
+                        distinctio consectetur voluptate magnam esse atque quia cumque sequi</p>
+                </div>
+                <div class="pricing-plan__info">
+                    <div class="pricing-plan__content">
+                        <div class="pricing-plan__content__top">
+                            <p class="pricing-plan__content__title">Free Trail</p>
+                            <p class="pricing-plan__content__subtitle"><span class="pricing-plan__content__little">$</span> 00
+                                <span class="pricing-plan__content__little">per month</span></p>
+                        </div>
+                        <p class="pricing-plan__content__text">30 Free Trail</p>
+                        <p class="pricing-plan__content__text">5 Free Projects</p>
+                        <p class="pricing-plan__content__text">PHP 5 Enabled</p>
+                        <p class="pricing-plan__content__text">24/7 Suports</p>
+                        <a href="#" class="pricing-plan__content__button">Order Now</a>                      
```

Comment:
Need to fix problem with jumping blocks (remove border probably)

Reply:
done